nx-X11/programs/Xserver/GL/mesa/GLcore/Imakefile

```diff
@@ -1,12 +1,7 @@
 XCOMM $XFree86: xc/programs/Xserver/GL/mesa/GLcore/Imakefile,v 1.3 2002/05/31 18:12:36 dawes Exp $
 
 #if DoLoadableServer
-#if !BuildModuleInSubdir
```

Comment:
Hm, wait. Why are you removing `BuildModuleInSubdir` logic? It's still defined (though to `NO`) in `nx-X11/config/cf/xfree86.cf` and `nx-X11/config/cf/xorg.cf` and also used in `nx-X11/programs/Xserver/fb/Imakefile`.